package-lock.json

```diff
@@ -1,3924 +1,8 @@
 {
 	"name": "~TODO~",
 	"version": "0.0.1",
-	"lockfileVersion": 2,
```

Comment:
I might drop this change from the PR. It looks like you're still on npm 6 and should upgrade to npm 7